fs/build

```diff
@@ -110,6 +110,23 @@ case "$ROOTFS" in
 		wget https://ci.ubports.com/job/xenial-mainline-edge-rootfs-$ARCH_DEB/lastSuccessfulBuild/artifact/out/ubuntu-touch-xenial-edge-$ARCH_DEB-rootfs.tar.gz
 		tar xfp ubuntu-touch-xenial-edge-$ARCH_DEB-rootfs.tar.gz -C $CADMIUMROOT/tmp/root
 	;;
+	fedora)
+		echo "Installing Fedora Linux"
+		# TODO: stop hardcoding this image
+		FEDORA_XZ="https://download.fedoraproject.org/pub/fedora-secondary/releases/35/Spins/aarch64/images/Fedora-Minimal-35-1.2.aarch64.raw.xz"
+		if [ ! -f "$CADMIUMROOT/tmp/fedora-$ARCH_UNAME.raw.xz" ]; then
+		wget $FEDORA_XZ -O $CADMIUMROOT/tmp/fedora-$ARCH_UNAME.raw.xz
+
+		fi
+		xz -v -k -d $CADMIUMROOT/tmp/fedora-$ARCH_UNAME.raw.xz
+		ROOTFS_LOOP=$(losetup -f -P fedora-$ARCH_UNAME.raw --show) # setup loop device
```

Comment:
isn't there a better way of doing this?
feels somewhat like a hack. perhaps there's just tar archive of rootfs somewhere